app/controllers/spotlight/catalog_controller.rb

```diff
@@ -126,15 +126,11 @@ def manifest
     protected
 
     def attach_breadcrumbs
-      # The "q: ''" is necessary so that the breadcrumb builder recognizes that a path like this:
-      # /exhibits/1?f%5Bgenre_sim%5D%5B%5D=map&q= is not the same as /exhibits/1
-      # Otherwise the exhibit breadcrumb won't be a link.
-      # see http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-current_page-3F
       if view_context.current_page?({ action: :admin })
-        add_breadcrumb t(:'spotlight.exhibits.breadcrumb', title: @exhibit.title), exhibit_root_path(@exhibit, q: '')
+        add_breadcrumb t(:'spotlight.exhibits.breadcrumb', title: @exhibit.title), exhibit_root_path(@exhibit)
```

Comment:
Comment by @cbeer from original PR #2656.
> The comment above this code suggested it was a necessary workaround, at least at some point in time. We should check if it's still an issue, or if it's fixed, also fix this on line 131, remove the comment, and pin the dependencies as appropriate.

Reply:
The code and comment were added when Rails 4 was required.  I was not able to reproduce the issue described in the comment since Rails 4 is no longer allowed and other dependencies also prevent using Rails 4.  I tested with Rails 5.2.  Both URLs were generated correctly.

RE: Pin - current Spotlight pins to `'rails', '>= 5.2', '< 6.2'`.  I believe this pin is sufficient based on testing with Rails 5.2.